htdocs/class/vendor/snoopy.php

```diff
@@ -997,5 +997,3 @@ function getResults()
         return $this->results;
     }
 }
-
-?>
```

Comment:
Please revert this change. This is a vendor supplied file (hence the new 'vendor' folder,) and it is easier to track if it is not changed.